src/main/java/TaskManager.java

```diff
@@ -0,0 +1,99 @@
+import java.util.ArrayList;
+
+public class TaskManager {
+
+    public static final String TASK_ALREADY_SET_ALERT = "Task was already set as done";
+    public static final String TASK_MARKED_MESSAGE = "Nice! I've marked this task as done:";
+    public static final String LIST_TASKS_MESSAGE = "Here are the tasks in your list:";
+    public static final String LIST_EMPTY_MESSAGE = "The list is empty";
+    public static final String TASK_ADDED_MESSAGE = "Got it. I've added this task:";
+
+    // Stores all the tasks provided
+    ArrayList<Task> tasks = new ArrayList<Task>();
+
+    // Adds a new task with the descriptionWithDetails provided by the user
+    public void addTask(TaskType taskType, String descriptionWithDetails){
+        switch (taskType) {
+        case ToDo:
+            tasks.add(new ToDo(descriptionWithDetails));
+            break;
+        case Deadline:
+            tasks.add(new Deadline(descriptionWithDetails));
+            break;
+        case Event:
+            tasks.add(new Event(descriptionWithDetails));
+            break;
+        }
+        printLatestTaskAndTotalNumberOfTasks();
```

Comment:
Function names can be shorter and more concise

Reply:
Will take note